nacc/redcap2nacc.py

```diff
@@ -146,6 +146,69 @@ def check_for_bad_characters(field: Field) -> typing.List:
     return incompatible
 
 
+def check_redcap_event(options, record) -> bool:
+    """
+    Determines if the record's redcap_event_name and filled forms match the
+    options flag
+    """
+    if options.lbd and options.ivp:
+        event_name = 'initial_visit'
+        form_match_lbd = record['lbd_ivp_b1l_complete']
+        if (form_match_lbd == '0' or form_match_lbd == ''):
+            event_match = False
+            return event_match
```

Comment:
Consider simplifying these returns.
```suggestion
            return False
```

Reply:
this makes the code look much cleaner, thanks